### ISSUE TITLE
first of all stop being a retard

### ISSUE THREAD

**Issue by simonboiii** [TT-Tutorials/Rage-Multi-Tool#2]
Another skidded shit of yours? you just fucking re designed astraahome multi tool with this shit imagine saying I fucking skidded raptor when I only helped aran code it plus you can't fucking code even if I asked you python basics questions you will just fucking google it like a normal skid plus why tf did you reply back with "lmao whats ur discord username n btw its funny how you say this is skidded when you have raptor multi tool in your ting and ur saying "new project coming soon" making me laugh" and "n its taken me 4 days to code the gen L" 4 days? a fucking https token gen only takes 23 minutes did you take 4 days designing your skidded piece of shit and doing no work? how retarded can you fucking get mr smartass

**Comment by VerstreuteSeele** [comment]
agreed but astraa is a disgrace to the dev community as hes a skid himself @simonboiii

**Comment by exotic8259** [comment]
> Another skidded shit of yours? you just fucking re designed astraahome multi tool with this shit imagine saying I fucking skidded raptor when I only helped aran code it plus you can't fucking code even if I asked you python basics questions you will just fucking google it like a normal skid plus why tf did you reply back with "lmao whats ur discord username n btw its funny how you say this is skidded when you have raptor multi tool in your ting and ur saying "new project coming soon" making me laugh" and "n its taken me 4 days to code the gen L" 4 days? a fucking https token gen only takes 23 minutes did you take 4 days designing your skidded piece of shit and doing no work? how retarded can you fucking get mr smartass

my boi whats ur discord?
